fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractExportTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * (C) Copyright IBM Corp. 2016, 2019, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.test.common;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Function;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.config.FHIRRequestContext;
+import com.ibm.fhir.model.resource.Basic;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.model.test.TestUtil;
+import com.ibm.fhir.model.type.Code;
+import com.ibm.fhir.model.type.Coding;
+import com.ibm.fhir.model.type.Element;
+import com.ibm.fhir.model.type.Extension;
+import com.ibm.fhir.model.type.Integer;
+import com.ibm.fhir.model.type.Meta;
+import com.ibm.fhir.persistence.ResourcePayload;
+
+/**
+ * Tests related to the high-speed export method in FHIRPersistence.
+ */
+public abstract class AbstractExportTest extends AbstractPersistenceTest {
+    Basic resource1;
+    Basic resource2;
+    Basic resource3;
+    Basic resource4;
+
+    @BeforeClass
+    public void createResources() throws Exception {
+        FHIRRequestContext.get().setTenantId("all");
+
+        Basic resource = TestUtil.readExampleResource("json/ibm/minimal/Basic-1.json");
+
+        Basic.Builder resource1Builder = resource.toBuilder();
+        Basic.Builder resource2Builder = resource.toBuilder();
+        Basic.Builder resource3Builder = resource.toBuilder();
+        Basic.Builder resource4Builder = resource.toBuilder();
+
+        // number
+        resource1Builder.extension(extension("http://example.org/integer", Integer.of(1)));
+        resource2Builder.extension(extension("http://example.org/integer", Integer.of(2)));
+        resource3Builder.extension(extension("http://example.org/integer", Integer.of(3)));
+        resource4Builder.extension(extension("http://example.org/integer", Integer.of(4)));
+
+        // save them in-order so that lastUpdated goes from 1 -> 3 as well
+        resource1 = persistence.create(getDefaultPersistenceContext(), resource1Builder.meta(tag("pagingTest")).build()).getResource();
+        resource2 = persistence.create(getDefaultPersistenceContext(), resource2Builder.meta(tag("pagingTest")).build()).getResource();
+        resource3 = persistence.create(getDefaultPersistenceContext(), resource3Builder.meta(tag("pagingTest")).build()).getResource();
+        resource4 = persistence.create(getDefaultPersistenceContext(), resource4Builder.meta(tag("pagingTest")).build()).getResource();
```

Comment:
not sure where "pagingTest" comes from; maybe update to "exportTest"?  or probably the tag isn't needed at all since you don't ever need to check them